generator/package.json

```diff
@@ -11,7 +11,7 @@
     "node": ">=12.0.0"
   },
   "dependencies": {
-    "gulp": "^4.0.2",
+    "gulp": "^5.0.0",
```

Comment:
Only concern is that this major bump dropped Node <10.13 support, but our package is marked as supporting Node >=10.3, so this would technically be a breaking change

Reply:
This a is "sub-package" for the purpose of generating samples. It has nothing to do with the main SDK/distributed bundle